jetty-server/src/main/java/org/eclipse/jetty/server/session/DatabaseAdaptor.java

```diff
@@ -257,6 +259,26 @@ public String getConnectionUrl()
         return _connectionUrl;
     }
 
+    public String getUsername()
```

Comment:
I don't see why we're adding these parameters? The jdbc url usually contains any authentication information that is necessary.

Reply:
correct but It's only me who don't like username/password in url :)

Reply:
The thing is, when this is configured by the session-store-jdbc.mod, the username and password will all be in config files anyway!